packages/synapse-react-client/src/lib/containers/PageProgress.tsx

```diff
@@ -45,28 +45,44 @@ const PageProgress: React.FunctionComponent<PageProgressProps> = props => {
     }
   }
 
+  const buttonSx = {
+    padding: '10px 120px',
+    marginRight: '30px',
+  }
+  const progressSx = {
+    backgroundColor: 'transparent',
+    '& .MuiLinearProgress-bar': {
+      backgroundColor: barColor,
+    },
```

Comment:
Allow LinearProgress to be a non-MUI color. Consider adding [SYNAPSE_GREEN](https://github.com/Sage-Bionetworks/SynapseWebClient/blob/cb96e14812334eb322ba4a08ca059788cb57c6e8/src/main/java/org/sagebionetworks/web/shared/WebConstants.java#L73) as a MUI color when handling PORTALS-2653